documentation/asciidoc/topics/proc_configuring_anti_affinity.adoc

```diff
@@ -0,0 +1,16 @@
+[id='configuring_anti_affinity-{context}']
+= Configuring Anti-Affinity
+Specify where {k8s} schedules pods for your {brandname} clusters to ensure availability.
+
+You can use the , preferring one over the other.
```

Comment:
I think you're missing a word or two  :slightly_smiling_face:

Reply:
bah. fixed.